x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table.tsx

```diff
@@ -320,6 +320,15 @@ export class FeatureTable extends Component<Props, {}> {
         options={options}
         idSelected={`${feature.id}_${selectedPrivilegeId ?? NO_PRIVILEGE_VALUE}`}
         onChange={this.onChange(feature.id)}
+        legend={i18n.translate('xpack.security.management.editRole.featureTable.actionLegendText', {
+          defaultMessage: '{featureName} privilege',
```

Comment:
This legend text would be a bit more accurate / less prone to cause confusion in the context of the rest of the page
```suggestion
          defaultMessage: '{featureName} feature privilege',
```

Reply:
Updated in 4401f4d